extension/js/common/core/attachment.ts

```diff
@@ -64,6 +66,71 @@ export class Attachment {
     this.contentDescription = contentDescription || undefined;
   }
 
+  public static prepareFileAttachmentDownload = async (attachment: Attachment, parentTabId: string) => {
+    const blacklistedFiles = [
```

Comment:
@rrrooommmaaa sir, I think I'm doing wrong to put this reusable function here since `js/common/core...` and `test/common/core../` codes are the same but have a conflict with paths - https://flowcrypt.semaphoreci.com/jobs/50cb70e7-f815-4e0b-9291-54bd864a5544. May I have your suggestion on this?